client/stylesheets/_modal.scss

```diff
@@ -283,7 +481,182 @@
 	END END END END END END
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	Add Listing Modal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+/*~~~~ Button ~~~~*/
+
+.headerDropDownAddListing {
+	float: left !important;
+	padding: 0 !important;
+	width: 91% !important;
+}
 
+.headerDropDownAddListing::before {
+	margin: -32px 8px 0 !important;
+}
+
+.modAddListingTrigger {
+	background-color: rgba(255, 255, 255, 0) !important;
+	color: #f8504b !important;
+	font-size: 0.9rem !important;
+	padding: 18px 50px 18px 0px !important;
+	width: 100% !important; 
+}
+
+.modAddListingTrigger i {
+	color: #f8504b !important;
+	margin-right: 10px !important;
+	margin-left: -25px !important;
+}
+
+/*~~ Button End ~~*/
+
+.modAddListingPageThree, .modAddListingPageFive, .modAddListingPageSix {
+	margin-top: 50px;
+}
+
+.modMultiBtnAddListingOne {
+	margin-top: 77px !important;
+}
+
+.modMultiBtnAddListingTwo {
+	margin-top: 103px;
+}
+
+.modMultiBtnAddListingThree {
+	margin-top: 76px;
+}
+
+.modMultiBtnAddListingFour {
```

Comment:
Selector `modMultiBtnAddListingFour` should be written in lowercase with hyphens